test/libsolidity/SemanticTest.cpp

```diff
@@ -192,7 +192,105 @@ vector<SideEffectHook> SemanticTest::makeSideEffectHooks() const
 				return result;
 			}
 			return {};
-		}};
+		},
+		bind(&SemanticTest::eventSideEffectHook, this, _1)
+	};
+}
+
+string SemanticTest::formatParameter(optional<AnnotatedEventSignature> _signature, bool _indexed, size_t _index, bytes const& _data)
+{
+	auto isPrintableASCII = [](string const& s)
```

Comment:
Why not define it for `bytes` if you're only ever using it with that type and the function cannot be used anywhere else?
```suggestion
	auto isPrintableASCII = [](bytes const& _s)
```